src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreJob.cs

```diff
@@ -140,20 +145,14 @@ internal sealed class SolutionRestoreJob : ISolutionRestoreJob
             _nuGetProjectContext.OperationId = request.OperationId;
             _isSolutionLoadRestore = isSolutionLoadRestore;
 
-            using (var ctr1 = token.Register(() => _status = NuGetOperationStatus.Cancelled))
+            try
             {
-                try
-                {
-                    await RestoreAsync(request.ForceRestore, request.RestoreSource, token);
-                }
-                catch (OperationCanceledException) when (token.IsCancellationRequested)
-                {
-                }
-                catch (Exception e)
-                {
-                    // Log the exception to the console and activity log
-                    await _logger.LogExceptionAsync(e);
-                }
+                await RestoreAsync(request.ForceRestore, request.RestoreSource, token);
+            }
+            catch (Exception e)
```

Comment:
Wouldn't this change cause cancellations to be added to the activity log?
Is that something we want to do?

Reply:
I would like to see the logs but, since this could change actual user experience, I decided not to show the token cancellation exception in the activity log.